README.md

```diff
@@ -137,6 +140,62 @@ A curated list of awesome [Jupyter](http://jupyter.org/) projects, libraries and
 * [ResGuides: research with Jupyter](https://www.gitbook.com/book/dansand/resguides-research-with-jupyter/details)
 * [Zero to JupyterHub](http://zero-to-jupyterhub.readthedocs.io/en/latest/) - Tutorial to help install and manage JupyterHub.
 
+## Jupyter Notebook, JupyterHub, JupyterLab
+- Jupyter Organization
+  -
+  - Homepage: https://jupyter.org/
+  - Homepage: https://jupyter.github.io/
+  - Community: https://jupyter.org/community
+    - Help: https://github.com/jupyter/help
+    - Mailing List: https://groups.google.com/forum/#!forum/jupyter
+    - Mailing List: https://groups.google.com/forum/#!forum/jupyter-education
+    - Mailing List: https://groups.google.com/forum/?pli=1#!forum/jupyter-research-facilities
+    - Chat: https://gitter.im/jupyter/jupyter
+    - Q&A: http://stackoverflow.com/questions/tagged/jupyter
+  - Source: https://github.com/jupyter
+  - Docs: https://jupyter.org/documentation
+  - Docs: **https://jupyter.readthedocs.io/en/stable/**
+  - Jupyter Governance
+    https://github.com/jupyter/governance
+    - JEP: Jupyter Enhancement Proposal
+      - https://github.com/jupyter/enhancement-proposals
+  - Blog: https://blog.jupyter.org/
+  - Twitter: [@ProjectJupyter](https://twitter.com/ProjectJupyter)
+- Jupyter Notebook
+  -
+  - Source: https://github.com/jupyter/notebook
+  - Docs: https://jupyter-notebook.readthedocs.io/en/stable/
+  - https://github.com/jupyter/jupyter/wiki/A-gallery-of-interesting-Jupyter-Notebooks
+- IPython
```

Comment:
Most of the IPython section looks rather stale. The [last IPEP is from 2014](https://github.com/ipython/ipython/wiki/IPEPs:-IPython-Enhancement-Proposals) and the [wiki](http://github.com/ipython/ipython/wiki) looks quite dated too. With the [split](https://blog.jupyter.org/the-big-split-9d7b88a031a7) of the Jupyter and IPython projects, the IPython parts seem less relevant to me. What do you think?